README.md

```diff
@@ -125,5 +164,5 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
```

Comment:
can you elaborate? The is responsible for the defines the **structure** of our database. This means that the schema will define all the tables and each column on each table in our database